src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/header.html.twig

```diff
@@ -0,0 +1,81 @@
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+<div class="product-header col-md-12">
+  <div class="row justify-content-md-center">
+  {% if isMultiShopContext %}
+    <div class="col-xxl-10">
+      <div class="alert alert-warning" role="alert">
+        <p class="alert-text">{{ 'You are in a multistore context: any modification will impact all your shops, or each shop of the active group.'|trans({}, 'Admin.Catalog.Notification') }}</p>
+      </div>
+    </div>
+  {% endif %}
+
+    <div class="col-xxl-10">
+      <div class="row">
+        <div class="col-md-8">
+          {{ form_widget(productForm.basic.name) }}
+          {{ form_errors(productForm.basic.name) }}
+        </div>
+        <div class="col-sm-7 col-md-2">
+          {{ form_widget(productForm.basic.type) }}
+          {{ form_errors(productForm.basic.type) }}
+        </div>
+        <div class="toolbar col-sm-3 col-md-2 text-md-right">
+          {% if null != statsLink %}
+          <a class="toolbar-button btn-sales" href="{{ statsLink }}" target="_blank" title="{{ 'Sales'|trans({}, 'Admin.Global') }}"
+            id="product_form_go_to_sales">
+            <i class="material-icons">assessment</i>
+            <span class="title">{{ 'Sales'|trans({}, 'Admin.Global') }}</span>
+          </a>
+          {% endif %}
+
+          <a
+            class="toolbar-button btn-quicknav btn-sidebar"
+            href="#"
+            title="{{ 'Quick navigation'|trans({}, 'Admin.Global') }}"
+            data-toggle="sidebar"
+            data-target="#right-sidebar"
+            data-url="{{ path('admin_product_list', {limit: 'last', offset: 'last', view: 'quicknav'}) }}"
+            id="product_form_open_quicknav"
+          >
+            <i class="material-icons">list</i>
+            <span class="title">{{ 'Product list'|trans({}, 'Admin.Catalog.Feature') }}</span>
+          </a>
+
+          <a class="toolbar-button btn-help btn-sidebar" href="#"
+            title="{{ 'Help'|trans({}, 'Admin.Global') }}"
+            data-toggle="sidebar"
+            data-target="#right-sidebar"
+            data-url="{{ helpLink }}"
+            id="product_form_open_help"
+          >
+            <i class="material-icons">help</i>
+            <span class="title">{{ 'Help'|trans({}, 'Admin.Global') }}</span>
+          </a>
```

Comment:
This looks like it can be reusable with an include and props too!

Reply:
Yeah same as the footer it's an ugly copy/paste from the product page v1 but it can definitely be improved
You are more than welcome to help us improve the product page layout, including some nice reusable small components 😉